tests/bench/benchfn.c

```diff
@@ -18,7 +18,7 @@
 #undef NDEBUG            /* assert must not be disabled */
 #include <assert.h>      /* assert */
 
-#include "timefn.h"        /* UTIL_time_t, UTIL_getTime */
+#include <timefn.h>        /* UTIL_time_t, UTIL_getTime */
```

Comment:
`timefn.h` is a project's header, not a system one.
It should use `""` double quotes

Reply:
Good point. I will fix that. I should just use relative paths, correct?

Reply:
correct